### PR TITLE
cindex: make pred np.array to allow negation

### PR DESCRIPTION
**Pull request by michalozeryflato**

### CODE REVIEW

File: fuse/eval/metrics/libs/survival.py [hunk 1]
```diff
@@ -22,6 +22,8 @@ def c_index(
         """
         if isinstance(pred[0], np.ndarray):
             pred = np.asarray(pred)[:, event_class_index]
+        elif isinstance(pred, list):
+            pred = np.asarray(pred)
```
Comment (mosheraboh) [inline fuse/eval/metrics/libs/survival.py:26]: pred is a list of what? floats? 
Should we add an assert that event_class_index is -1?

Comment (michalozeryflato) [reply fuse/eval/metrics/libs/survival.py:26]: pred should contain float score per sample.
The ground truth has two components: event/censor indicator and time of event/censor

Comment (michalozeryflato) [reply fuse/eval/metrics/libs/survival.py:26]: @mosheraboh see new fix - following your comments

**Review by mosheraboh — APPROVED** [review]
Looks good

**Comment by mosheraboh** [comment]
@michalozeryflato , the PR is waiting for you to merge.

**Pull request merged**